tests/api/admin/controller/test_sitewide_registration.py

```diff
@@ -172,7 +172,7 @@ def test_sitewide_registration_post_success(self):
         )
 
         # A registration document with an encrypted secret
-        shared_secret = binascii.hexlify(os.urandom(24))
+        shared_secret = binascii.hexlify(os.urandom(8))
```

Comment:
Had to reduce the size of the shared secret in the tests, to reflect the smaller key being used for testing.